### ISSUE TITLE
Full IPv6 Support

### ISSUE THREAD

**Issue by TheMeinerLP** [CloudNetService/CloudNet-v2#275]
We need a complete IPv6 support for master and wrapper (workers in the future).
It is important to create multiple listeners for BungeeCord to listen on all IP addresses. And for the server it is important to listen on "*". That IPv6 is supported.
All this should be configurable for wrapper(worker) via config or command.

**Comment by GiantTreeLP** [comment]
>config or command.

How about both.

**Comment by TheMeinerLP** [comment]
Yes, I could. But the worker gets all the necessary information from the master, so it will be more likely to be set by command.
